tests/unit/dataclasses/test_template_generation.py

```diff
@@ -118,7 +118,7 @@ def test_csv_generation_inmemory(
         template = template.inline_dependencies()
     bindings, filled = template.fill(BLDG, include_optional=include_optional)
 
-    with NamedTemporaryFile(mode="w", suffix=".csv") as dest:
+    with NamedTemporaryFile(mode="w", suffix=".csv", delete=False) as dest:
```

Comment:
Should we have a `dest.flush()` in this test?

Reply:
I think there is already a dest.flush() in this test